<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClooneyScanner</class>
 <widget class="QMainWindow" name="ClooneyScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clooney Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>503</y>
      <width>180</width>
      <height>48</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reject the scan and get a new one.</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="scan_preview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>340</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filepath_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>340</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>504</y>
      <width>178</width>
      <height>48</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reject the scan and get a new one.</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertical_line_button">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Vertical Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four_corners_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Four Corners</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_180_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Rotate 180Ëš</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
